clients/venice-admin-tool/src/main/java/com/linkedin/venice/Arg.java

```diff
@@ -118,7 +118,10 @@ public enum Arg {
   VENICE_CLIENT_SSL_CONFIG_FILE(
       "venice-client-ssl-config-file", "vcsc", true, "Configuration file for querying key in Venice client through SSL."
   ), STARTING_OFFSET("starting_offset", "so", true, "Starting offset when dumping admin messages, inclusive"),
-  MESSAGE_COUNT("message_count", "mc", true, "Max message count when dumping admin messages"),
+  STARTING_TIMESTAMP(
```

Comment:
Can we use `START_DATE` arg?
```  START_DATE("start-date", "sd", true, "Start date in PST. Example: 2020-10-10 10:10:10"),```

Reply:
Updated to use `START_DATE` and `END_DATE` with topic dump